packages/core/scss/components/floating-label/_variables.scss

```diff
@@ -12,10 +12,20 @@ $kendo-floating-label-focus-scale: null !default;
 $kendo-floating-label-focus-offset-x: null !default;
 $kendo-floating-label-focus-offset-y: null !default;
 
+$kendo-floating-label-border-scale: null !default;
+$kendo-floating-label-border-height: null !default;
+$kendo-floating-label-border-offset-x: null !default;
+$kendo-floating-label-border-padding: null !default;
+
+$kendo-floating-label-inside-scale: null !default;
+$kendo-floating-label-inside-height: null !default;
+
 $kendo-floating-label-transition: null !default;
 
 $kendo-floating-label-bg: null !default;
 $kendo-floating-label-text: null !default;
 
 $kendo-floating-label-focus-bg: null !default;
 $kendo-floating-label-focus-text: null !default;
+
+$kendo-floating-label-border-bg: k-color( surface-alt ) !default;
```

Comment:
I think in the core variable files we should keep only _null_ values

Reply:
yes, agree, I'll update this now